docs/variants/protectli_vp66xx/post-install.md

```diff
@@ -0,0 +1,18 @@
+# Post-installation setup
+
+This document contains extra steps to perform after installing Dasharo in order
+to enable full functionality.
+
+## Windows
+
+This section covers Windows post-install steps tested on Windows 11. It is
+likely that similar procedures would for others Linux distributions as well.
+
+### Qualcomm Atheros cards may be missing driver
+
+If you experience Windows to fail to recognize Qualcomm Atheros WiFi cards,
+you may need to install the necessary driver manually.
+
+Download the Windows driver from https://www.ath-drivers.eu/download-driver-nr-382-for-atheros-QCA6174A-and-Windows10.html
```

Comment:
Can you use md syntax to hide the link? E.g. [Windows driver](link).

Reply:
Done.